src/main/java/com/uber/cadence/common/RetryOptions.java

```diff
@@ -27,10 +34,37 @@
 import java.util.Objects;
 
 public final class RetryOptions {
+  public static final RetryOptions DEFAULT_SERVICE_OPERATION_RETRY_OPTIONS;
 
+  private static final Duration RETRY_SERVICE_OPERATION_INITIAL_INTERVAL = Duration.ofMillis(20);
+  private static final Duration RETRY_SERVICE_OPERATION_EXPIRATION_INTERVAL = Duration.ofMinutes(1);
+  private static final double RETRY_SERVICE_OPERATION_BACKOFF = 1.2;
```

Comment:
I don't think these constants belong here. The retry options are used by workflow code that shouldn't really be aware about underlying RPC layer.